synapse/tests/test_lib_storm.py

```diff
@@ -337,6 +337,46 @@ def test_storm_deltag(self):
             self.none(node[1].get('#foo.bar'))
             self.none(node[1].get('#baz.faz'))
 
+    def test_storm_refs_ndef(self):
+        with self.getRamCore() as core:
+            pnode = core.formTufoByProp('ps:person', 32 * '0')
+            enode = core.formTufoByProp('inet:email', 'c00l@vertex.link')
+            pvalu = pnode[1].get('ps:person')
+            phas0 = core.formTufoByProp('ps:has', (pvalu, ('inet:email', 'c00l@vertex.link')))
+            core.formTufoByProp('ps:has', (pvalu, ('inet:fqdn', 'vertex.link')))
+
+            # It does not actually make sense to have a file reference a ps:has, this is just for testing
```

Comment:
that isn't necessarily true. imagine a file which is the scan of a document which denotes the possession of something by a person.

Reply:
OK, I'll remove the comment

Reply:
done